src/cluster/sharding.rst

```diff
@@ -0,0 +1,289 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _cluster/sharding:
+
+========
+Sharding
+========
+
+.. _cluster/sharding/scaling-out:
+
+Scaling out
+===========
+
+Normally you start small and grow over time. In the beginning you might do just
+fine with one node, but as your data and number of clients grows, you need to
+scale out.
+
+For simplicity we will start fresh and small.
+
+Start node1 and add a database to it. To keep it simple we will have 2 shards
+and no replicas.
+
+.. code-block:: bash
+
+    curl -X PUT "http://xxx.xxx.xxx.xxx:5984/small?n=1&q=2" --user daboss
+
+If you look in the directory ``data/shards`` you will find the 2 shards.
+
+.. code-block:: text
+
+    data
```

Comment:
How about to explicitly how here that data, shards and others are directories? Like:

```
data/
+-- shards/
     +-- 00000000-7fffffff/
     |   +-- small.1425202577.couch
     +-- 80000000-ffffffff/
         +-- small.1425202577.couch

```